README.md

```diff
@@ -13,6 +13,35 @@ bash install-devenv.sh
 **If you are setting up a new development environment, run `devenv bootstrap` after installing.**
 
 
+
+## what is it / technical overview
```

Comment:
Since this isn't a question, should we change it to: `what it is`?

Reply:
i'll change it to just `technical overview`